modules/nf-core/parabricks/fq2bam/main.nf

```diff
@@ -0,0 +1,73 @@
+process PARABRICKS_FQ2BAM {
+    tag "$meta.id"
+    label 'process_high'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used with Parabricks at the moment. Please use docker or singularity."
+    }
+
+    container "nvcr.io/nvidia/clara/clara-parabricks:4.0.0-1"
+
+    input:
+    tuple val(meta), path(reads), path(interval_file)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(index)
+    path known_sites
+    val markdups
+    val qc_metrics
```

Comment:
Unless I miss something these are used to set some args. This could also be handled in the modules.config directly leaving us with more flexibilty. Generally, no vals should be passed into modules but only files. --> users can overwrite arguments more easily.

Reply:
These are intended to be boolean vals to set args yes. 

`markdups` if false, the tool returns after sorting and doesn't mark duplicates or perform BQSR. If true, the tool marks duplicates and optionally performs BQSR. Since this controls both functionality and outputs, does it belong in modules.config? Or in the meta map (as arguments like single ended are there)? This is unclear to me.

`qc_metrics` is similar but less impactful. If true, a folder of qc metrics is calculated and returned.

Reply:
As @FriederikeHanssen mentioned already, both `markdups` and `qc_metrics` can go in `text.args` and don't belong in the module